scipy/stats/_multivariate.py

```diff
@@ -6978,3 +6980,305 @@ def entropy(self):
 
         """
         return self._dist._entropy(self.dim, self.kappa)
+
+
+class normal_inverse_gamma_gen(multi_rv_generic):
+    r"""Normal inverse gamma distribution.
+
+    The normal inverse gamma distribution is the conjugate prior of a normal
```

Comment:
```suggestion
    The normal-inverse-gamma distribution is the conjugate prior of a normal
```
Here and throughout.

Feel free to suggest (concise) introductory content.